src/components/Header/Header.styled.ts

```diff
@@ -35,3 +36,8 @@ export const MenuItems = styled(Box)`
 export const ClickableHeading = styled(Heading)`
   cursor: pointer;
 `;
+
+export const TextLogo = styled(DaviLogo)`
+  width: 40%;

```

Comment:
Using the width as a percentage here breaks the navbar on mobile. I think its good enough for now, but we should fix this later.

Reply:
If you want to set it to a percentage also use a max width.